packages/vaadin-themable-mixin/vaadin-themable-mixin.js

```diff
@@ -197,7 +205,7 @@ export const ThemableMixin = (superClass) =>
       super.finalize();
 
       const template = this.prototype._template;
-      if (!template || hasThemes(this.is)) {
+      if (!template || classHasThemes(this)) {
```

Comment:
One drawback is that since `finalize()` is run for each level in the mixin hierarchy, the guard no longer stops processing here if the themes were already processed for the same tag name earlier :/